serverless/event_sources/knative-event-sources.adoc

```diff
@@ -0,0 +1,30 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="knative-event-sources"]
+= Getting started with event sources
+include::modules/common-attributes.adoc[]
+:context: knative-event-sources
+toc::[]
+
+An event source is a resource object that acts as a link between an event producer and a _sink_. A sink can be a Knative Service, Channel, or Broker that receives Events from an event source.
```

Comment:
Is it a resource object? If serverless is built using the CRD API, it's probably a custom resource. Only core k8s objects are resources; Most other things are custom resources via the API.

Writing about these concepts is always a challenge, when determining what is reasonable to capitalize.

Reply:
If it's OK I'll leave this one for the follow up PRs to address - I'm currently doing work on the Eventing content overall, so I'd prefer not to block these PRs so that I can rebase my other work on top of them. I just wanted to get this file added for now so that this topicmap doesn't clash with master :slightly_smiling_face: